dashboard/config/locales/en.yml

```diff
@@ -1404,9 +1404,27 @@ en:
     founder: "Founder, Code.org"
     your_friends: "Your Friends at Code.org"
   lti:
-    create_integration_success: "Succesfully created your LTI integration"
+    create_integration_success: "Successfully created your LTI integration"
     error:
       integration_exists: "Lti Integration already exists for this Client ID"
-      missing_params: "Missing required param(s): client_id, lms, email"
+      missing_params: "Missing required param(s): School/District name, Client ID, LMS, Email"
       unsupported_lms_type: "Unsupported LMS platform type"
       wrong_resource_type: "Only LtiResourceLink is supported right now"
+    integration:
+      client_id: "LMS Client ID"
+      email: "Your email"
+      name: "School or district name?"
```

Comment:
Minor nit: the other field labels don't have question marks, except for the one that is a full question. IMO we should be consistent

Reply:
Good call. I'll remove it here. Will leave the question mark on the LMS drop down selector, if that feels good.

Reply:
Yeah I think it makes sense to keep it on the dropdown selector